src/Tasks/Common/Resources/xlf/Strings.de.xlf

```diff
@@ -262,11 +262,6 @@
         <target state="translated">NETSDK1046: Der TargetFramework-Wert "{0}" ist nicht gültig. Verwenden Sie für mehrere Ziele die Eigenschaft "TargetFrameworks".</target>
         <note>{StrBegin="NETSDK1046: "}</note>
       </trans-unit>
-      <trans-unit id="SkippingAdditionalProbingPaths">
-        <source>NETSDK1048: 'AdditionalProbingPaths' were specified for GenerateRuntimeConfigurationFiles, but are being skipped because 'RuntimeConfigDevPath' is empty.</source>
-        <target state="translated">NETSDK1048: Für GenerateRuntimeConfigurationFiles wurden "AdditionalProbingPaths" angegeben, sie werden jedoch übersprungen, weil "RuntimeConfigDevPath" leer ist.</target>
-        <note>{StrBegin="NETSDK1048: "}</note>
-      </trans-unit>
```

Comment:
@v-zbsail Do you know why this one was moved in a few files?

Reply:
It's not necessary to fix this now, but it would be helpful in the future if the order was always preserved by handback. It makes it easier to review and reduces merge conflicts.

Reply:
cc @tmeschter

Reply:
This trans unit is present in this strings.de.xlf. Seems order of string changed

Reply:
@nguerrera String is changed to different line in the same file.

Reply:
Yes. I'm asking to avoid moving it on handback from where it was at handoff.